src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/BalanceSheetServiceImpl.java

```diff
@@ -35,4 +35,18 @@ public BalanceSheet getBalanceSheet(String transactionId, String companyAccounts
 
         return balanceSheet;
     }
+
+    @Override
+    public void postBalanceSheet(String transactionId, String companyAccountsId, BalanceSheet balanceSheet)
+            throws ApiErrorResponseException {
+
+        ApiClient apiClient = apiClientService.getApiClient();
+
+        CurrentPeriodApi currentPeriod = transformer.getCurrentPeriod(balanceSheet);
+
+        apiClient.transaction(transactionId)
+                .companyAccount(companyAccountsId)
+                .smallFull()
+                .currentPeriod().create(currentPeriod);
```

Comment:
do you not care about any return values from the create call?

Reply:
Not particularly; if any errors occur an exception will be thrown and caught higher up anyway :)